packages/config/src/utils/extensions/utils.ts

```diff
@@ -32,6 +35,7 @@ export const installExtension = async ({
     }),
     headers: {
       'netlify-token': netlifyToken,
+      'User-Agent': extensionInstallSource,
```

Comment:
So the user agent will be `'buildbot' | 'cli' | 'require'`? For future-proofness, do we want to be a bit more descriptive and include the fact that we're calling from Netlify Build, potentially including the version?